centreon-ha/ocf-scripts/mariadb-centreon-common.sh

```
@@ -318,3 +336,92 @@ mysql_common_stop()
     return $OCF_SUCCESS
 
 }
+
+# $1: the message that'il be logged
+repli_log_ok()
+{
+    if [ ${OCF_RESKEY_enable_repli_eplanation} >= "1" ];then
```

Shellcheck (0.10.0)

[warning] OCF_RESKEY_enable_repli_eplanation is referenced but not assigned (did you mean 'OCF_RESKEY_enable_repli_explanation'?).

(SC2154)